examples/http-rust/spin.toml

```diff
@@ -1,4 +1,4 @@
-spin_version = "1"
+spin_manifest_version = "1"
```

Comment:
Will other examples be changed too, or will we defer that to a later PR?

Reply:
Yes, this should be reverted. Like you said, i think we want to do this all at once in a later PR